cloud/webfaction/webfaction_db.py

```diff
@@ -75,6 +75,19 @@
         description:
             - The machine name to use (optional for accounts with only one machine)
         required: false
+        
+    postgis:
+        description:
+            - Enable postgis addon for Postgres
+        choices: BOOLEANS 
```

Comment:
Hi,
This needs to be:

`choices: [ "true", "false" ]`

Which should fix the Travis Failure